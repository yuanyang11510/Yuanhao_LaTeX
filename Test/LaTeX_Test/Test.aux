\relax 
\@writefile{toc}{\contentsline {section}{\numberline {1}aaa}{1}{}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1}aaa}{4}{}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {2}bbb}{5}{}\protected@file@percent }
\gdef \@abspage@last{6}
