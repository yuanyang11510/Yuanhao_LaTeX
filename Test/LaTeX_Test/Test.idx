\indexentry{D}{2}
\indexentry{E}{2}
\indexentry{F}{2}
\indexentry{G}{2}
\indexentry{H}{2}
