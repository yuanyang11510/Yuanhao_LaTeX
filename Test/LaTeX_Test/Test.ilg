This is makeindex, version 2.17 [TeX Live 2024] (kpathsea + Thai support).
Scanning input file Test.idx....done (5 entries accepted, 0 rejected).
Sorting entries....done (11 comparisons).
Generating output file Test.ind....done (21 lines written, 0 warnings).
Output written in Test.ind.
Transcript written in Test.ilg.
