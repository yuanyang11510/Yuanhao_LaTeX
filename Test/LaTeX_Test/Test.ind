\begin{theindex}

  \item D, 2

  \indexspace

  \item E, 2

  \indexspace

  \item F, 2

  \indexspace

  \item G, 2

  \indexspace

  \item H, 2

\end{theindex}
