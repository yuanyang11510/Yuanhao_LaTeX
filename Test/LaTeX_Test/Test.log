This is XeTeX, Version 3.141592653-2.6-0.999996 (TeX Live 2024) (preloaded format=xelatex 2024.9.6)  12 SEP 2024 16:16
entering extended mode
 restricted \write18 enabled.
 file:line:error style messages enabled.
 %&-line parsing enabled.
**Test
(./Test.tex
LaTeX2e <2024-06-01> patch level 2
L3 programming layer <2024-08-16>
(c:/texlive/2024/texmf-dist/tex/latex/base/article.cls
Document Class: article 2024/02/08 v1.4n Standard LaTeX document class
(c:/texlive/2024/texmf-dist/tex/latex/base/size10.clo
File: size10.clo 2024/02/08 v1.4n Standard LaTeX file (size option)
)
\c@part=\count190
\c@section=\count191
\c@subsection=\count192
\c@subsubsection=\count193
\c@paragraph=\count194
\c@subparagraph=\count195
\c@figure=\count196
\c@table=\count197
\abovecaptionskip=\skip49
\belowcaptionskip=\skip50
\bibindent=\dimen141
) (c:/texlive/2024/texmf-dist/tex/latex/geometry/geometry.sty
Package: geometry 2020/01/02 v5.9 Page Geometry
 (c:/texlive/2024/texmf-dist/tex/latex/graphics/keyval.sty
Package: keyval 2022/05/29 v1.15 key=value parser (DPC)
\KV@toks@=\toks17
) (c:/texlive/2024/texmf-dist/tex/generic/iftex/ifvtex.sty
Package: ifvtex 2019/10/25 v1.7 ifvtex legacy package. Use iftex instead.
 (c:/texlive/2024/texmf-dist/tex/generic/iftex/iftex.sty
Package: iftex 2022/02/03 v1.0f TeX engine tests
))
\Gm@cnth=\count198
\Gm@cntv=\count199
\c@Gm@tempcnt=\count266
\Gm@bindingoffset=\dimen142
\Gm@wd@mp=\dimen143
\Gm@odd@mp=\dimen144
\Gm@even@mp=\dimen145
\Gm@layoutwidth=\dimen146
\Gm@layoutheight=\dimen147
\Gm@layouthoffset=\dimen148
\Gm@layoutvoffset=\dimen149
\Gm@dimlist=\toks18
) (c:/texlive/2024/texmf-dist/tex/latex/enumitem/enumitem.sty
Package: enumitem 2019/06/20 v3.9 Customized lists
\labelindent=\skip51
\enit@outerparindent=\dimen150
\enit@toks=\toks19
\enit@inbox=\box52
\enit@count@id=\count267
\enitdp@description=\count268
) (c:/texlive/2024/texmf-dist/tex/latex/blindtext/blindtext.sty
Package: blindtext 2012/01/06 V2.0 blindtext-Package
 (c:/texlive/2024/texmf-dist/tex/latex/tools/xspace.sty
Package: xspace 2014/10/28 v1.13 Space after command names (DPC,MH)
)
\c@blindtext=\count269
\c@Blindtext=\count270
\c@blind@countparstart=\count271
\blind@countxx=\count272
\blindtext@numBlindtext=\count273
\blind@countyy=\count274
\c@blindlist=\count275
\c@blindlistlevel=\count276
\c@blindlist@level=\count277
\blind@listitem=\count278
\c@blind@listcount=\count279
\c@blind@levelcount=\count280
\blind@mathformula=\count281
\blind@Mathformula=\count282
\c@blind@randomcount=\count283
\c@blind@randommax=\count284
\c@blind@pangramcount=\count285
\c@blind@pangrammax=\count286
) (c:/texlive/2024/texmf-dist/tex/latex/lipsum/lipsum.sty (c:/texlive/2024/texmf-dist/tex/latex/l3packages/l3keys2e/l3keys2e.sty (c:/texlive/2024/texmf-dist/tex/latex/l3kernel/expl3.sty
Package: expl3 2024-08-16 L3 programming layer (loader) 
 (c:/texlive/2024/texmf-dist/tex/latex/l3backend/l3backend-xetex.def
File: l3backend-xetex.def 2024-05-08 L3 backend support: XeTeX
\g__graphics_track_int=\count287
\l__pdf_internal_box=\box53
\g__pdf_backend_annotation_int=\count288
\g__pdf_backend_link_int=\count289
))
Package: l3keys2e 2024-08-16 LaTeX2e option processing using LaTeX3 keys
)
Package: lipsum 2021-09-20 v2.7 150 paragraphs of Lorem Ipsum dummy text
\g__lipsum_par_int=\count290
\l__lipsum_a_int=\count291
\l__lipsum_b_int=\count292
 (c:/texlive/2024/texmf-dist/tex/latex/lipsum/lipsum.ltd.tex)) (c:/texlive/2024/texmf-dist/tex/latex/fancyhdr/fancyhdr.sty
Package: fancyhdr 2024/07/23 v4.3.1 Extensive control of page headers and footers
\f@nch@headwidth=\skip52
\f@nch@O@elh=\skip53
\f@nch@O@erh=\skip54
\f@nch@O@olh=\skip55
\f@nch@O@orh=\skip56
\f@nch@O@elf=\skip57
\f@nch@O@erf=\skip58
\f@nch@O@olf=\skip59
\f@nch@O@orf=\skip60
) (./Test.aux)
\openout1 = `Test.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 26.
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 26.
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 26.
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 26.
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for TS1/cmr/m/n on input line 26.
LaTeX Font Info:    Trying to load font information for TS1+cmr on input line 26.
 (c:/texlive/2024/texmf-dist/tex/latex/base/ts1cmr.fd
File: ts1cmr.fd 2023/04/13 v2.5m Standard LaTeX font definitions
)
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for TU/lmr/m/n on input line 26.
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 26.
LaTeX Font Info:    ... okay on input line 26.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 26.
LaTeX Font Info:    ... okay on input line 26.

*geometry* driver: auto-detecting
*geometry* detected driver: xetex
(c:/texlive/2024/texmf-dist/tex/latex/base/atbegshi-ltx.sty
Package: atbegshi-ltx 2021/01/10 v1.0c Emulation of the original atbegshi
package with kernel methods
)
*geometry* verbose mode - [ preamble ] result:
* driver: xetex
* paper: <default>
* layout: <same size as paper>
* layoutoffset:(h,v)=(0.0pt,0.0pt)
* modes: showframe 
* h-part:(L,W,R)=(92.14519pt, 430.00462pt, 92.14519pt)
* v-part:(T,H,B)=(95.39737pt, 556.47656pt, 143.09605pt)
* \paperwidth=614.295pt
* \paperheight=794.96999pt
* \textwidth=430.00462pt
* \textheight=556.47656pt
* \oddsidemargin=19.8752pt
* \evensidemargin=19.8752pt
* \topmargin=-13.87262pt
* \headheight=12.0pt
* \headsep=25.0pt
* \topskip=10.0pt
* \footskip=30.0pt
* \marginparwidth=65.0pt
* \marginparsep=11.0pt
* \columnsep=10.0pt
* \skip\footins=9.0pt plus 4.0pt minus 2.0pt
* \hoffset=0.0pt
* \voffset=0.0pt
* \mag=1000
* \@twocolumnfalse
* \@twosidefalse
* \@mparswitchfalse
* \@reversemarginfalse
* (1in=72.27pt=25.4mm, 1cm=28.453pt)



[1

]

[2]

[3]

[4]

[5]

[6] (./Test.aux)
 ***********
LaTeX2e <2024-06-01> patch level 2
L3 programming layer <2024-08-16>
 ***********
 ) 
Here is how much of TeX's memory you used:
 2247 strings out of 474235
 41861 string characters out of 5745819
 1936612 words of memory out of 5000000
 24960 multiletter control sequences out of 15000+600000
 558117 words of font info for 42 fonts, out of 8000000 for 9000
 1348 hyphenation exceptions out of 8191
 79i,5n,93p,211b,153s stack positions out of 10000i,1000n,20000p,200000b,200000s

Output written on Test.pdf (6 pages).
