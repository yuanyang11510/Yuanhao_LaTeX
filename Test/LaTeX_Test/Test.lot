\contentsline {table}{\numberline {1}{\ignorespaces text\footnotemark }}{2}{table.1}%
\contentsline {table}{\numberline {2}{An optional table caption (used in the list of tables)}}{2}{table.2}%
