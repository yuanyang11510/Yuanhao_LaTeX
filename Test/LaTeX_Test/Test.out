\BOOKMARK [1][-]{section.1}{\376\377\000A\000A\000A}{}% 1
